src/Fetcher/ConfTechFetcher.php

```diff
@@ -32,136 +34,206 @@ public function __construct(RegistryInterface $doctrine, LoggerInterface $logger
         $this->repository = $this->em->getRepository(Conference::class);
     }
 
-    public function getUrl(array $params): string
+    public function getUrl(array $params = []): string
     {
         return "https://raw.githubusercontent.com/tech-conferences/conference-data/master/conferences/$params[date]/$params[tag].json";
     }
 
     public function fetch(): array
     {
-        $params = [
-            '2018' => [
-                'php',
-                'javascript',
-                'golang',
-            ],
-            '2019' => [
-                'php',
-                'ruby',
-                'css',
-            ],
-        ];
+        $params = $this->matchTags();
 
         $conferencesFinal = [];
+        $newConferencesCount = 0;
 
         foreach ($params as $date => $technologies) {
             foreach ($technologies as $technologie) {
                 $client = new Client();
```

Comment:
The Client should be initialized outside of the loops, to avoid reinitialization